test: TestTxnWriteFail fails

```
=== RUN TestTxnWriteFail
...
timeout after 10minutes
```

I still can reproduce this on my machine

```
2016-03-09 10:40:33.864896 I | rafthttp: the connection with 1934a5a62e5ad643 became inactive
2016-03-09 10:40:33.925102 E | rafthttp: failed to dial 1934a5a62e5ad643 on stream MsgApp v2 (dial tcp 127.0.0.1:20247: getsockopt: connection refused)
2016-03-09 10:40:33.925140 I | rafthttp: the connection with 1934a5a62e5ad643 became inactive
2016/03/09 10:40:34 grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial unix localhost:node4140295217575820494.sock: connect: no such file or directory"; Reconnecting to "localhost:node4140295217575820494.sock"
2016/03/09 10:40:36 grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial unix localhost:node4140295217575820494.sock: connect: no such file or directory"; Reconnecting to "localhost:node4140295217575820494.sock"
2016-03-09 10:40:37.819036 W | etcdserver: failed to reach the peerURL(http://127.0.0.1:20247) of member 1934a5a62e5ad643 (Get http://127.0.0.1:20247/version: dial tcp 127.0.0.1:20247: getsockopt: connection refused)
2016-03-09 10:40:37.819068 W | etcdserver: cannot get the version of member 1934a5a62e5ad643 (Get http://127.0.0.1:20247/version: dial tcp 127.0.0.1:20247: getsockopt: connection refused)
2016/03/09 10:40:38 grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial unix localhost:node4140295217575820494.sock: connect: no such file or directory"; Reconnecting to "localhost:node4140295217575820494.sock"
2016/03/09 10:40:38 grpc: Conn.transportMonitor exits due to: grpc: timed out trying to connect
--- FAIL: TestTxnWriteFail (229.01s)
	txn_test.go:51: timed out waiting for txn to fail
```

--- a/clientv3/integration/txn_test.go
+++ b/clientv3/integration/txn_test.go
@@ -45,8 +45,9 @@ func TestTxnWriteFail(t *testing.T) {
 		donec <- struct{}{}
 	}()
 
+	dialTimeout := 5 * time.Second
 	select {
-	case <-time.After(5 * time.Second):
+	case <-time.After(2*dialTimeout + time.Second):
 		t.Fatalf("timed out waiting for txn to fail")
 	case <-donec:
 		// don't restart cluster until txn errors out

